src/multirust-cli/multirust_mode.rs

```diff
@@ -326,7 +326,7 @@ fn add_target(cfg: &Cfg, m: &ArgMatches) -> Result<()> {
         pkg: "rust-std".to_string(),
         target: TargetTriple::from_str(target),
     };
-    try!(toolchain.add_component(new_component));
+    try!(toolchain.add_component(new_component, &cfg));
```

Comment:
That makes sense. The only thing `Cfg` is used for in `log_telemetry` is to get the path of the `multirust_dir` for logging. Would it be more reasonable to pass a `PathBuf` through as some kind of log_destination parameter?